Doc/library/asyncio-task.rst

```diff
@@ -276,6 +276,9 @@ Sleeping
    If *result* is provided, it is returned to the caller
    when the coroutine completes.
 
+   ``sleep()` always suspends the current task, allowing other tasks
```

Comment:
The function always suspends the current task, allowing other tasks to run.

Reply:
Amended the commit. BTW should it be "the coroutine" rather than "the function"? The previous paragraph uses "the coroutine" referring to `sleep()`.

Reply:
I actually liked "``sleep()` always suspends the current task" **way more** than "the function always...".

Reply:
I think we should revert this PR back the original phrasing and merge it.

Reply:
I've now reverted the PR to the original phrasing.